base_import_pdf_by_template/wizards/wizard_base_import_pdf_upload.py

```diff
@@ -122,6 +122,19 @@ def action_process(self):
         self.attachment_id.write({"res_model": record._name, "res_id": record.id})
         return record
 
+    def _add_log_error_text(self, field_name, value):
```

Comment:
This change seems unrelated

Reply:
This change is to avoid adding code that repeats things, similar to what is suggested at https://github.com/OCA/edi/pull/1051#discussion_r1805914480.

Reply:
Yeah, but it's unrelated to the other change, so please put it in a different commit.